tests/test_web.py

```
@@ -910,3 +910,222 @@ def test_export_includes_score_breakdown(self, client, settings):
+
+        msg = "Settings saved successfully."
+        result = toast_header(msg, "success")
+        parsed = json.loads(result["HX-Trigger"])
+        assert parsed["toast"]["message"] == msg
+
+
+class TestToastIntegration:
+    """Tests that toast headers appear on mutation endpoint responses."""
+
+    @patch("option_alpha.web.routes._run_scan_task")
+    def test_scan_trigger_has_toast_header(self, mock_scan, client, settings):
```

Ruff (0.15.0)

[warning] Unused method argument: `mock_scan`

(ARG002)